src/wast-parser.cc

```diff
@@ -604,7 +610,7 @@ bool WastParser::ParseVarListOpt(VarVector* out_var_list) {
 Result WastParser::ParseValueType(Type* out_type) {
   WABT_TRACE(ParseValueType);
   if (!PeekMatch(TokenType::ValueType)) {
-    return ErrorExpected({"i32", "i64", "f32", "f64", "v128"});
+    return ErrorExpected({"i32", "i64", "f32", "f64", "v128", "anyref", "anyfunc"});
```

Comment:
anyfunc (i.e. funcref) is not allowed as a value type in this proposal (it was split out).